src/renderer/components/ProviderAccordion.js

```diff
@@ -56,7 +56,7 @@ const ProviderAccordion = ({ keys, onNewSshKeyClicked, onActionClicked }) => {
               keys.map(key => (
                 <div
                   className="w-48 h-48 relative rounded-lg shadow-lg bg-gray-200 text-center mt-4"
-                  key={`${key.provider}-${key.path}`}>
+                  key={`${key.path}`}>
```

Comment:
Path should be unique for each ssh key so it makes sense to define key prop using just path parameter.

Reply:
Yup, it does! Glad we kept it like this to and were able to find out other issue related to it.